### ISSUE TITLE
Used kernel for WarpCRANSACflow

### ISSUE THREAD

**Issue by Camilochiang** [PruneTruong/DenseMatching#23]
Hei! Great code and organized github! Thanks for your work ;)
I'm using your weights to align images using RANSAC-flow. The only problem that I have is that i'm getting some deformation in the images. I look around your code and in your WarpC paper bud didn't find any detail about the size of the used kernel. In ransac-flow was 7. Did you use the same size?

Thanks!

**Comment by PruneTruong** [comment]
Hi, thanks for your interest in our work! I think I used kernel 7 for RANSAC-Flow, along with all default parameters. What kind of deformations are you seeing? Did you try reproducing the results of the paper with the provided weights?

**Comment by Camilochiang** [comment]
Yes. Got same or really similar results, but when testing in my images, there are deformations
For example: This is the target image (To which one I want to align)
![IMG_000X__1](https://user-images.githubusercontent.com/48471579/195016653-cb8595ef-f580-4b81-b2c3-d95aa06f4fc4.jpg)

Here the raw image to be aligned (to the previous image)
![Screenshot from 2022-10-11 08-54-54](https://user-images.githubusercontent.com/48471579/195017689-b88dbf68-c436-49e0-862a-df6209cdb53c.png)

And after the alignment process
![IMG_000X__3](https://user-images.githubusercontent.com/48471579/195016628-9ac7bbd8-20f8-41dc-b69a-c5c04e81ad85.jpg)

there are some image deformations ... 

Thanks!

**Comment by PruneTruong** [comment]
Hi, this indeed sounds pretty weird, almost like it is shifted in the wrong direction. I do not have access to RANSAC-Flow codebase anymore, but did you check how 1) RANSAC-Flow original is doing, 2) the first stage homography is doing. If the homography is failing, then there is no way the network can recover since it assumes small displacement. Otherwise, is there any particular reason you want to use RANSAC-Flow? PDC-Net should be doing ok on these kind of examples i think.
